docs/pages/quick-start.mdx

```diff
@@ -124,7 +123,7 @@ Each system can contain any number of methods, though this sytem only has the si
 
 ## Client package
 
-The client package will vary depending on which template used (vanilla or react), but much of it will remain the same between the two. Vite is used for bundling and serving the client app. As mentioned earlier there is boilerplate code included in each to bootstrap MUD. There are a few files you should understand and be aware of as you build your project:
+The client package will vary depending on which template used (vanilla, react or phaser), but much of it will remain the same between the two. Vite is used for bundling and serving the client app. As mentioned earlier there is boilerplate code included in each to bootstrap MUD. There are a few files you should understand and be aware of as you build your project:
```

Comment:
I'm an oxford comma fan
```suggestion
The client package will vary depending on which template used (vanilla, react, or phaser), but much of it will remain the same between the two. Vite is used for bundling and serving the client app. As mentioned earlier there is boilerplate code included in each to bootstrap MUD. There are a few files you should understand and be aware of as you build your project:
```